eessi_bot_job_manager.py

```diff
@@ -253,7 +253,7 @@ def determine_finished_jobs(self, known_jobs, current_jobs):
 
     def read_job_pr_metadata(self, job_metadata_path):
         """
-        Read job metadata file and return the contents of the 'PR' section.
+        Determine metadata od a job
```

Comment:
This function now only calls another function, so shouldn't we just remove it and replace all calls to it by a call to `read_job_metadata_from_file`?

Reply:
Good idea. Implemented in https://github.com/EESSI/eessi-bot-software-layer/pull/247/commits/7ed3d8d32b24da271bbd0ac0e7b513f1e6ee2b9e

Moved corresponding test from `tests/test_eessi_bot_job_manager.py` to `tests/test_tools_job_metadata.py`.